packages/design-system/src/styles/components/_Table.scss

```diff
@@ -1,13 +1,18 @@
 @import '../settings/index.scss';
 
+$table-header-background-color: $color-gray-lightest !default;
+$table-stripe-background-color: $color-gray-lightest !default;
+$table-stripe-header-background-color: $color-primary-alt-lightest !default;
+$table-stacked-title-max-width: 160px;
```

Comment:
Is this referring to the max width of the header when the table is stacked? could we rename this and add a comment explaining what it is?
`table-stacked-header-max-width`